test/test_python_dispatch.py

```diff
@@ -205,6 +222,23 @@ def test_format(self) -> None:
         self.assertEqual(s1, s2)
         self.assertEqual(s1, s3)
 
+    def test_finite_tensor(self) -> None:
+        x = FiniteTensor(torch.ones(1))
+        self.assertRaises(AssertionError, lambda: x / torch.zeros(1))
+
```

Comment:
what happens if an in-place operation is performed on a `FiniteTensor` and it causes the tensor values to no longer be finite? (in this case: `div_`)

Reply:
just saw you already have a TODO for that above! nice